frontend/projects/valtimo/layout/src/lib/components/widget-management/management-action-button/widget-management-action-button.component.scss

```
@@ -25,9 +23,9 @@
     width: 100%;
   }
 
-  &__type .cds--list-box,
-  &__form .cds--text-input {
-    background: var(--cds-layer-02);
+  ::ng-deep &__type .cds--list-box,
```

Stylelint (17.11.1)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)